docs/source/tutorials/experimentation/exp_plotting.rst

```diff
@@ -0,0 +1,149 @@
+     num_at_level81: [23, 19, 14, 13, 19, 21, 15, 24, 17, 13, 20, 18, 19, 18, 20, 16, 13, 15, 22, 17]
+   [MOBSTER-RELU]:
+     num_at_level1: [241, 319, 352, 438, 354, 386, 197, 262, 203, 387, 320, 139, 359, 401, 334, 294, 361, 403, 178, 141]
+     num_at_level3: [110, 156, 135, 166, 138, 143, 104, 124, 95, 136, 133, 71, 133, 151, 130, 122, 134, 151, 92, 74]
+     num_at_level9: [50, 83, 59, 75, 59, 55, 57, 72, 53, 53, 58, 40, 62, 63, 61, 54, 52, 65, 48, 47]
+     num_at_level27: [31, 51, 29, 31, 29, 23, 39, 38, 36, 20, 29, 36, 32, 29, 32, 29, 24, 27, 31, 34]
+     num_at_level81: [20, 35, 12, 11, 12, 15, 22, 18, 26, 12, 16, 27, 16, 15, 20, 15, 15, 13, 18, 22]
+
+* There are four subfigures arranged as two-by-two matrix. Each contains two
+  curves in bold, along with confidence intervals. The horizontal axis
+  depicts wall-clock time, and on the vertical axis, we show the best
+  metric value found until this time.
+* More general, the data from our 640 experiments can be grouped w.r.t.
+  subplot, then *setup*. Each setup gives rise to one curve (bold, with
+  confidence band). Subplots are optional, the default is to plot a single
+  figure.
+* The function ``metadata_to_setup`` maps the metadata stored for an experiment
+  to the setup name. In our basic case, the setup is simply the name of the
+  method.
+* The function ``metadata_to_subplot`` maps the metadata to the subplot
+  index (0, 1, 2, 3). We group setups with the same configuration space,
+  but also split multi-fidelity and single-fidelity methods.
+* Once the data is grouped w.r.t. benchmark, then subplot (optional), then
+  setup, we should be left with 20 experiments, one for each seed. These 20
+  curves are now interpolated to a common grid, and at each grid point, the
+  20 values are aggregated into ``lower``, ``aggregate``, ``upper``. In the
+  figure, ``aggregate`` is shown in bold, and ``lower``, ``upper`` in dashed.
+  Different aggregation modes are supported (selected by
+  ``plot_params.aggregate_mode``).
+* We pass ``extra_results_keys`` to the
+  :meth:`~syne_tune.experiments.ComparativeResults.plot` method in order to
+  also retrieve extra results. This method returns a dictionary, whose
+  "extra_results" entry is what we need.
+
+Advanced Experimenteering
+-------------------------
+
+Once you start to run many experiments, you will get better at avoiding
+wasteful repetitions. Here are some ways in which Syne Tune can support you.
+
+* Combining results from several studies: It often happens that results for
+  a new idea need to be compared to baselines on a common set of benchmarks.
+  You do not have to re-run baselines, but can easily combine older results
+  with more recent ones. This is explained
+  `here <../visualization/comparisons.html#combining-sesults-from-multiple-studies>`__.
+* When running many experiments, some will fail. Syne Tune supports you in not
```

Comment:
> some will fail

Not actionable for this PR, but I'd be interested to learn where the failures come from, and if there's anything we can do about them. (Syne Tune problem, or SageMaker problem, or user error, or other?)

Reply:
I'll change to "may fail".

Reply:
This is indeed quite rare. But I've even seen things where dataset files could not be downloaded.

Another thing happening to me is that experiments for a method are failing, because there is a bug, but they work for all others.

If you are very careful and always test everything locally etc., this is unlikely. But sometimes, a bug only appears quite later.